src/main/java/org/terasology/launcher/settings/LauncherSettingsValidator.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 MovingBlocks
```

Comment:
Update the year

Reply:
Thanks for pointing that out. I updated the year to 2019.